src/components/Container/Container.tsx

```diff
@@ -37,15 +37,19 @@ export interface ContainerProps<T extends ElementType> {
   orientation?: Orientation;
   padding?: PaddingOptions;
   wrap?: WrapOptions;
+  fillHeight?: boolean;
+  maxHeight?: string;
+  minHeight?: string;
```

Comment:
where do we need these?

maybe we can create a more specific component that handles only this? 

cc @gjones

Reply:
I added this inorder to have similar props as the width so its fillWidth, minWidth, maxWidth and fillHeight, minHeight, maxHeight